model/src/test/kotlin/AdvisorRecordTest.kt

```diff
@@ -37,7 +37,7 @@ class AdvisorRecordTest : WordSpec({
     "hasIssues" should {
         "return false given the record has no issues" {
             val record = advisorRecordOf(
-                langId to listOf(createResult(1))
+                langId to listOf(createResult())
```

Comment:
Commit message: "Use a default value of 1 for the index in tests for which its value does not matter." (This better explains that you're acutally introducing a default value and not using an existing one.)